packages/angular_devkit/core/src/json/schema/registry.ts

```diff
@@ -339,14 +469,16 @@ export class CoreSchemaRegistry implements SchemaRegistry {
         valid: true,
         compile: (schema, _parentSchema, it) => {
           const compilationSchemInfo = this._currentCompilationSchemaInfo;
-          if (!compilationSchemInfo) {
+          if (compilationSchemInfo === undefined) {
             throw new Error('Invalid JSON schema compilation state');
+          } else if (compilationSchemInfo === null) {
+            return () => true;
```

Comment:
You can collapse these into the second conditional block since not having the info object is now valid in a non-compilation scenario.

Reply:
Doesn't work (see e2e errors). I thought I looked at that code path already.